src/sagas/authSaga.js

```diff
@@ -0,0 +1,26 @@
+import { put, call, takeLatest } from "redux-saga/effects";
+import { actionTypes } from "../actions/types";
+import authService from "../service/authService";
+import {
+  loginSuccess,
+  loginError,
+  loginRequest,
+  getErrors,
+} from "../actions/authActions";
+
+function* loginSaga(action) {
+  try {
+    const { email, password } = action.payload;
```

Comment:
Here you destruct object And in second line you create same object.
Why you just don't use action.payload